examples/demo.rs

```diff
@@ -1,24 +1,34 @@
 /* in your Cargo.toml
 [dependencies]
-twitchchat = "0.10"                                        # this crate
-tokio = { version = "0.2", features = ["full", "macros"] } # you need tokio to run it
+# this crate
+twitchchat = "0.10"
+
+# and now for tokio
+# macros allows you to use `#[tokio::main]` and `tokio::pin!` and `tokio::select!`
+# rt-multi gives you a multi-threaded runtime.
+tokio = { version = "0.2", features = ["rt-multi", "macros"] }
```

Comment:
Tokio has no `rt-multi` feature.

From docs.rs/tokio
> `rt-core`: Enables `tokio::spawn` and the basic (single-threaded) scheduler.
`rt-threaded`: Enables the heavier, multi-threaded, work-stealing scheduler.
`rt-util`: Enables non-scheduler utilities.

Reply:
Yeah, it should be tr-threaded. I'll need to change all of those comments when I bump the version so it'll read v0.11.0 anyway. Thanks for reminding me. I always get it wrong (i.e. is it rt-thread, rt-multi, rt-threaded?)